examples/aes/src/aes.rs

```diff
@@ -18,6 +20,10 @@ bytes!(Bytes144, 144);
 bytes!(Bytes176, KEY_SCHEDULE_LENGTH);
 bytes!(Key128, BLOCKSIZE);
 
+type ByteSeqResult = Result<ByteSeq, u8>;
+type BlockResult = Result<Block, u8>;
+type WordResult = Result<Word, u8>;
```

Comment:
I guess this style is not too ugly ?

Reply:
Yeah with the type alias this works nicely and having to rewrite the type is less painful.